src/promptflow/promptflow/_sdk/data/executable/main.py.jinja2

```diff
@@ -9,7 +9,7 @@ from bs4 import BeautifulSoup, NavigableString, Tag
 
 from promptflow._sdk._utils import print_yellow_warning
 from promptflow._sdk._serving.flow_invoker import FlowInvoker
-from promptflow._utils.multimedia_utils import is_multimedia_dict, MIME_PATTERN
```

Comment:
Please do not change the function name here, I have changed the function name in multimedia_utils.

Reply:
revert